integration/autoscalinggroup.go

```diff
@@ -43,11 +43,16 @@ func (group AutoScalingGroup) GetTargetInstances(canonical semver.Version, minim
     healthy,
     unhealthy)
 
-  if len(healthy) <= minimumInstanceCount {
-    fmt.Printf("%s => no action taken, not enough healthy instances\n", group.Name)
+	if len(healthy) <= minimumInstanceCount {
+    fmt.Printf("%s => not enough healthy instances\n", group.Name)
     return []string{}, nil
   }
 
+	if len(healthy) != len(group.InstanceDetails) {
```

Comment:
Could there be an issue here?

e.g. given instances A, B, C is it possible to have

Healthy = A, B
Details = B, C

?